### ISSUE TITLE
@stylistic/selector-combinator-space-before gives false positive when combined with comments

### ISSUE THREAD

**Issue by tbradsha** [stylelint-stylistic/stylelint-stylistic#66]
### What minimal example or steps are needed to reproduce the bug?

```
.some_class {
	.blue,
	// There's a comment here.
	+ .green,
	// And another.
	> .purple {
		background: yellow;
	}
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "customSyntax": "postcss-scss",
  "plugins": "@stylistic/stylelint-plugin",
  "rules": {
		"@stylistic/selector-combinator-space-before": "always"
  }
}
```

### How did you run Stylelint?

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIB0BnCBbGB9MAbAhiigATAA6AdmQC5IBGuArjADSU0D0HxAKgBYwATjADkJfMUjpMFasQHCk7agGpiSAObCYFNlWpdiAQQpRi+ChGoKl+gHzqADo0GPcMUspp18YANZaEIym8MQAnjC4uBAA7gDcygC+lIkgLCAAZgCW7gBy+JgIIDAAHgVuMKhgRGngEBTZGkXkFMTEZOCMKNQYAMphsvglHaEdjhDd1SgAtChTHXptY0waWRQoI+0gAALdYbhZ3VlgHHvuB7LTboyrVGmUSyCCjO4bCJ76NB271PuH1MdTpEYGAeoJplI6Gt8GDZo5fDBpnQYBkIMJNh18LgYvgwm8HsRkhQibVIA0shoAGJo9AwooAKzQFFqsEcKGaBLGE2oU1m83eHQI1Bg3QWnJAZ0ia2oGJAQpFMvurS2Pz+RxOkou1CuKzWsvlopAKRAiSAA)

### Which Stylelint-related dependencies are you using?

See demo and above.

### What did you expect to happen?

There is no violation.

### What actually happened?

The linter claims a violation (and in the second instance, the position is odd).
